packages/firestore/exp/api.ts

```diff
@@ -64,6 +64,12 @@ export {
   SetOptions,
   DocumentData,
   UpdateData,
+  Primitive,
+  WithFieldValue,
+  NestedUpdateFields,
+  AddPrefixToKeys,
+  NestedPartial,
+  UnionToIntersection,
```

Comment:
Optional: I think it would be cleaner if these were exports from a dedicated module, which would make it easier to understand that `SetOptions` and `Primitive` are completely unrelated.

Reply:
Are you saying to move the helper types into a separate `exp/helper.ts` file, or for `lite` as well?

Reply:
Something like lite/types.ts that you can refer to from exp/ and lite/

Reply:
Moved into `lite/types`